apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -171,7 +182,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
                 else:
                     html_params["incorrect"] = True
             except Exception as exc:
-                raise ValueError("invalid score" + score) from exc
+                raise ValueError(f"invalid score: {score}") from exc
```

Comment:
This error message is not really needed, and the above code block can be replaced by using https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/python/prairielearn.py#L140

Reply:
Changed in [e2b032b](https://github.com/PrairieLearn/PrairieLearn/pull/11188/commits/e2b032b9f9b129183820f07273bd4532213d0ecd).